### ISSUE TITLE
Support for Slurm 23.02?

### ISSUE THREAD

**Issue by mwharvey521** [natefoo/slurm-drmaa#76]
We recently updated our HPC cluster and Galaxy server, and slurm-drmaa no longer seems to work properly. Is there any chance of getting it updated to work with the most recent version of Slurm - 23.02?

**Comment by IBEXCluster** [comment]
Dear @mwharvey521 
Is it possible to give it a try this workaround: 
```
export DRMAA_LIBRARY_PATH=lib.slurm-22.05.6/libdrmaa.so.1
export LD_LIBRARY_PATH=lib.slurm-22.05.6:$LD_LIBRARY_PATH
```
This workaround is used as a temporary solution for our regular operation which uses slurm-drmaa. :-)

**Comment by mwharvey521** [comment]
Thanks for the response. I'm not sure what setting those variables is 
trying to point to. We are running slurm-23.02.01 on a Rocky 8 Linux system:

***@***.*** ~]$ export 
DRMAA_LIBRARY_PATH=lib.slurm-22.05.6/libdrmaa.so.1
***@***.*** ~]$ export 
LD_LIBRARY_PATH=lib.slurm-22.05.6:$LD_LIBRARY_PATH
***@***.*** ~]$ 
/software/apps/slurm-drmaa/current/bin/drmaa-run testscript.sh
F #3b67c3 [     0.00]  * Could not load DRMAA library 
(DRMAA_LIBRARY_PATH=lib.slurm-22.05.6/libdrmaa.so.1)
F #3b67c3 [     0.00]  * Error


Thanks,

Mike

**Comment by IBEXCluster** [comment]
Dear Mike,
I was referring the lib.slurm-22.05.6/ is from the old slurm library. However, please refer the master repo. Thanks to @wickhagj and he applied the patches for Slurm 23.x.x. Please pull the latest and recompile the slurm-drmaa. 

https://github.com/natefoo/slurm-drmaa/pull/79/commits/1f5db98cd788677e7a94b93cb56bc00266539fe2

**Comment by natefoo** [comment]
Merged in #79 and will be included in the 1.1.4 release.
